src/components/CustomScrollbar/TrackedScrollbar.tsx

```diff
@@ -0,0 +1,49 @@
+import {rem} from 'polished';
+import React from 'react';
+import Scrollbars from 'react-custom-scrollbars';
+import {dims} from '../styled';
+
+const defaultHeight = `calc(100% - ${rem(dims.tileHeaderHeight)}`;
+
+export interface TrackedScrollbarProps {
+  styles?: any;
+  setLastScrollPosition: (position: number) => void;
+  getLastScrollPosition: () => number;
+}
+
+class TrackedScrollbar extends React.Component<TrackedScrollbarProps> {
+  scrollbar: any;
+
+  constructor(props: TrackedScrollbarProps) {
+    super(props);
+  }
+
+  onScrollStop = () => {
+    if (this.props.setLastScrollPosition) {
+      this.props.setLastScrollPosition(this.scrollbar.getScrollTop());
```

Comment:
Shouldn't it take scroll value from the scrollbar? It's strange that external components would return position of scrollbar... I would recommend you save that value inside scrollbar and made some interface for getter, setter is redundant